tests/testthat/test-hover_reload_button.R

```diff
@@ -0,0 +1,33 @@
+context("Reload functionality")
+
+test_that("Test for three components in the button. Type is a 'button'.",
+          "'onClick' is present in the list. Javascript code is present", {
+
+  # Create testing input
+  test_me <- hover_reload_button("id", "label")
+
+  # Test 1: attribs type is a button
+  expect_match(
+    ## actual
+    test_me$attribs$type,
+    ## expect
+    "button"
+  )
+
+  # Test 2: names on attribs list contains onClick
+  expect_match(
```

Comment:
I think `expect_true(any("onClick" %in% names(test_me$attribs)))` would work well here.